w06h02/test/pgdp/ds/MultiStackTest.java

```diff
@@ -227,7 +227,5 @@ void popShouldReturnCorrectValues() {
         assertEquals(value3, stack.pop());
         assertEquals(value4, stack.pop());
         assertEquals(value2, stack.pop());
-
-        assertArrayEquals(array(0), getMem(getHead()));
```

Comment:
Would it make sense to add another of the Min_Value tests here, e.g. validating that after usage the stack has correctly been identified as empty? That would be a better replacement for the current tests, wouldn't it?

Reply:
Yes that's a good idea. So we could just check if top/pop returns `Integer.MIN_VALUE`.